### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by Minyewoo**

### CODE REVIEW

**Comment by a-givertzman** [comment]
- [ ] divide all tests into separate files

File: test/unit/core/entities/ds_data_class_test.dart [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:hmi_core/hmi_core.dart';
+
+void main() {
+  group('DsDataClass', () {
```
Comment (a-givertzman) [inline test/unit/core/entities/ds_data_class_test.dart:5]: add test to verify that class has all necessary names with proper values:
  requestAll('requestAll'),
  requestList('requestList'),
  requestTime('requestTime'),
  requestAlarms('requestAlarms'),
  requestPath('requestPath'),
  syncTime('syncTime'),
  commonCmd('commonCmd'),
  commonData('commonData');

File: test/unit/core/entities/ds_data_type_test.dart [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:hmi_core/hmi_core.dart';
+
+void main() {
+  group('DsDataType', () {
```
Comment (a-givertzman) [inline test/unit/core/entities/ds_data_type_test.dart:5]: add test to verify that class has all necessary names with proper values:
  bool('bool', 1),
  integer('int', 2),
  uInt('uint', 2),
  dInt('dint', 4),
  word('word', 2),
  lInt('lint', 8),
  real('real', 4),
  time('time', 4),
  dateAndTime('date_and_time', 8);

File: test/unit/core/entities/ds_status_test.dart [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:hmi_core/hmi_core.dart';
+
+void main() {
+  group('DsStatus', () {
```
Comment (a-givertzman) [inline test/unit/core/entities/ds_status_test.dart:5]: add test to verify that class has all necessary names with proper values:
  ok(0),
  obsolete(2),
  timeInvalid(3),
  invalid(10);

File: test/unit/core/entities/ds_data_type_test.dart (hunk 2 above)
Comment (Minyewoo) [reply test/unit/core/entities/ds_data_type_test.dart]: Done

File: test/unit/core/entities/ds_data_class_test.dart (hunk 1 above)
Comment (Minyewoo) [reply test/unit/core/entities/ds_data_class_test.dart]: Done

**Comment by Minyewoo** [comment]
> [ ] divide all tests (groups where not possible to split group into separate tests) into separate files

Done

**Review by a-givertzman — APPROVED** [review]

**Pull request merged**